src/components/Table/Table.scss

```diff
@@ -33,9 +33,25 @@ $block: '.#{variables.$ns}styled-table';
         border-block-end: 1px solid var(--g-color-line-generic);
     }
 
-    &__row:last-of-type {
-        #{$block}__cell {
-            border-block-end: none;
+    &__row {
+        &:last-of-type {
```

Comment:
Why not `last-child`?

Reply:
It was already there, I just reformatted it because of addition of &_vertical-align block :)